### DISCUSSION TITLE
Duplicate detection broken?

### DISCUSSION THREAD

**Discussion by MacPaille** [firefly-iii/firefly-iii#5667] [Firefly III] [answered]
Hey @JC5 

Anything changed that broke detection of duplicates in latest 5.6.13? All my transactions were imported with FIDI where almost all should have been duplicates...

Thanks!

**Comment by JC5** [comment, marked as answer]
No, not that I know of, and I've seen no duplicates myself.

**Reply by MacPaille** [reply]
No idea what happened. I have deleted all my older transactions and kept only the new one. After a new run they are all found as duplicate so no new transactions are added.

Really don't know what happened...
